### PR TITLE
Arc test

### PR DESCRIPTION
**Pull request by im-alexandre**
Included a method to calculate mantissas's coordinates (x, y) and plot graphic

### CODE REVIEW

**Comment by milcent** [comment]
Sounds nice! 
Looking forward to testing it.

File: benford/benford.py [hunk 1]
```diff
@@ -947,6 +949,26 @@ def show_plot(self, figsize=(15, 8)):
         plt.legend(loc='upper left')
         plt.show()
 
+def arc_test(mant, decimals = 2, grid=True, figsize=10):
```
Comment (milcent) [inline benford/benford.py]: Could you insert a docstring?

**Review by milcent — COMMENTED** [review]
Working fine.
What do you think of adding it also as a method to the Mantissa class?

**Comment by im-alexandre** [comment]
I do agree. Think It's better than a static method. I'm going to add the docstring and turn this into a Mantissas Class method.

**Comment by im-alexandre** [comment]
Done!

**Comment by milcent** [comment]
Closing this PR, since everything was solved on
[PR-25](https://github.com/milcent/benford_py/pull/25)

**Comment by milcent** [comment]
Version 0.2.0 on PyPi.
Try it!
Cheers

Em qua, 2 de out de 2019 às 17:12, im-alexandre <notifications@github.com>
escreveu:

**Pull request closed without merging**